docs/widgets.md

```diff
@@ -13,27 +11,89 @@ Widgets can:
 * Allow users to quickly access pieces of the app to complete key or regular tasks (e.g. Course Services, My Professional Development, Course Guide)
 * Provide users with at-a-glance information that represents the main use for the widget (e.g. Weather)
 
-### Advantages of widget types over custom widgets
+## Basic widgets
+
+The barebones widget provides an app title, a large icon, and a launch button with configurable text. It's a simple link to an app or external URL.
+
+![basic widget](./img/basic-widget.png)
+
+### Sample entity file
+
```

Comment:
Or link to a full-sized example file included with the docs?  Like, here's the most interesting snippet, and linked is the full example?